mayastor-test/test_nexus.js

```diff
@@ -320,16 +429,20 @@ describe('nexus', function () {
       assert.equal(nexus.children[0].state, 'CHILD_ONLINE');
       assert.equal(nexus.children[1].uri, `aio:///${aioFile}?blk_size=4096`);
       assert.equal(nexus.children[1].state, 'CHILD_ONLINE');
+
       assert.equal(
         nexus.children[2].uri,
-        `iscsi://${externIp}:3261/iqn.2019-05.io.openebs:disk1`
-      );
-      assert.equal(nexus.children[2].state, 'CHILD_ONLINE');
-      assert.equal(
-        nexus.children[3].uri,
         'nvmf://127.0.0.1:8420/nqn.2019-05.io.openebs:disk2'
       );
-      assert.equal(nexus.children[3].state, 'CHILD_ONLINE');
+      assert.equal(nexus.children[2].state, 'CHILD_ONLINE');
+      if (doIscsiReplica) {
+        assert.equal(
+          nexus.children[2].uri,
```

Comment:
Shouldn't the index be 3? doIscsiReplica will also affect the index used by uring below, I think.

Reply:
Yup, pushing a fix now.